quickwit-indexing/src/source/mod.rs

```diff
@@ -161,7 +162,7 @@ pub fn quickwit_supported_sources() -> &'static SourceLoader {
     })
 }
 
-#[derive(Clone)]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
```

Comment:
PartialEq might be a little ambitious... You'd be surprised by the semantics of PartialEq of a json object in general, and `serde_json::Value` in particular.